packages/reflect-yjs/src/awareness.test.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import {expect, test, beforeEach, suite, afterEach} from 'vitest';
+import {Awareness} from './awareness.js';
+import {resolver} from '@rocicorp/resolver';
+
+import {Mutators, mutators} from './mutators.js';
+import * as Y from 'yjs';
+
+import {Reflect} from '@rocicorp/reflect/client';
```

Comment:
Generally keep the imports together without newlines in between. Then VSCode can sort/organize them automatically